api/src/main/resources/workbench-api.yaml

```diff
@@ -6548,6 +6571,57 @@ definitions:
     properties:
       egressEvent:
         "$ref": "#/definitions/EgressEvent"
+  AuditEgressEventRequest:
+    type: object
+    description: Placeholder request object for future extension.
+  AuditEgressRuntimeLogEntry:
+    type: object
+    properties:
+      timestamp:
+        type: string
+        description: Time at which the log message was written, in ISO 8601 format.
+      message:
+        type: string
+        description: The log message text payload.
+  AuditEgressRuntimeLogGroup:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Display name for this log group
+      pattern:
+        type: string
+        description: Pattern used to match the log message text
+      entries:
+        type: array
+        description: >
+          The log entries matching the specified pattern, in descending timestamp
+          order. May be truncated.
+        items:
+          "$ref": "#/definitions/AuditEgressRuntimeLogEntry"
+      totalEntries:
+        type: integer
+        description: >
+          The total number of log entries matching this pattern. To determine
+          whether entries are truncated, compare against len(entries).
+  AuditEgressEventResponse:
```

Comment:
Are we planing to put all those external links in the response or you want to let UI construct them?

Reply:
My plan was to let the UI construct these

Reply:
That SG. Then we might need `apiBaseUrl` in getConfig response.